examples/machine-learning/build-hcs-images/README.md

```diff
@@ -0,0 +1,63 @@
+# Build images for HCS (Hypercompute cluster service) cluster
+
+This document describes steps to deploy images for cluster deployed using
+Cluster Director (HCS). For each image it may take upto ~60 min to finish the
+build process and publish image in your project before you move ahead with
+deploying cluster.
+
+## Requirements
+
+Make sure `gcluster`, `gcloud`, `terraform` and `packer`
+binaries to be available in `PATH`.
+
+## Build A3M image
+
+In order to build image for A3 mega machine, follow these steps,
+
+```sh
+cd cluster-toolkit/examples/machine-learning/build-hcs-images/
+PROJECT=<your-project-id>
+SUFFIX=<image-family>
+./build.sh a3m $SUFFIX $PROJECT
+```
+
+## Build common image
+
+In HCS cluster, common image is used in GPU machines/ non-GPU machines and
+login nodes. you can build this image by following,
+
+```sh
+cd cluster-toolkit/examples/machine-learning/build-hcs-images/
+PROJECT=<your-project-id>
+SUFFIX=<image-family>
+./build.sh common $SUFFIX $PROJECT
+```
+
+## Next Steps
+
+After the images are published in your project. You can choose these images
```

Comment:
Delete all content from this section and instead say:

"The two images that were build can be found under xxx family. Use the following gcloud command to describe the images and confirm they were built:"

And then give the two gcould commands.